BCStudentSoftwareDevTeam/celts
# Auto Populate Event Location

fixes PR #755 

Did: 
- added a column called 'programLocation' to Program. 
- On program management page, we added a 'location' column so that admin can update the location of a program. 
- Populate the location for an event of the event creation page to be the same as the location set for a program
- fixed tests

Test: 
- reset database and run tests. no test should fail
- go to Admin>Settings>Program Managment: Set a location for a program
- create event for that program from above. On the create event page you should see the location has been populated with what you had entered for the program location.
